firmware/README.asciidoc

```diff
@@ -2,13 +2,14 @@
 
 The mainboard receives structs in the form:
 
+
 typedef struct Command {

```

Comment:
It would be better to put code into code block.
https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/